### PR TITLE
change to computeHashSlotOrReplyError to not reply error

### PR DESCRIPTION
**Pull request by sjpotter**
move error returning to caller

based on feedback from @tezc

### CODE REVIEW

File: src/redisraft.c [hunk 1]
```diff
@@ -418,7 +418,11 @@ static RRStatus handleSharding(RedisRaftCtx *rr,
 {
     int slot;
 
-    if (computeHashSlotOrReplyError(rr, ctx, cmds, &slot) != RR_OK) {
+    if (computeHashSlot(rr, cmds, &slot) != RR_OK) {
+        if (ctx) {
```
Comment (tezc) [inline src/redisraft.c:422]: At this point, we always have a `ctx`, don't we?

**Comment by tezc** [comment]
New commit is a bit different than the previous commit and PR title :) Is it intentional or by mistake? Also, this pr will merge into master, just fyi.

**Comment by tezc** [comment]
@sjpotter Did you see the comment above?

**Comment by tezc** [comment]
@sjpotter Did you see the comment above?

File: src/redisraft.c (hunk 1 above)
Comment (sjpotter) [reply src/redisraft.c:422]: I guess I was thinking that it may be called by a follower, though perhaps thats impossible

**Comment by sjpotter** [comment]
> @sjpotter Did you see the comment above?

removed extra check

**Comment by tezc** [comment]
> New commit is a bit different than the previous commit and PR title :) Is it intentional or by mistake? Also, this pr will merge into master, just fyi.

I meant this one.

**Comment by sjpotter** [comment]
I meant this to go to both.  figured this was simple enough to go straight into master.

**Comment by sjpotter** [comment]
oh, and yes, that was a mistake.  that shouldn't have been here.

**Comment by sjpotter** [comment]
was a rebase mistake. I rebased against wrong tree and then pushed to the pr against master

**Review by tezc — APPROVED** [review]

**Pull request merged**